index.html

```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <nav id="navbar">
+      <header class="js_doc">JS Documentation</header>
+      <ol>
+        <li><a class="nav-link" href="#Introduction">Introduction</a></li>
+        <li>
+          <a class="nav-link" href="#What_you_should_already_know"
+            >What you should already know</a
+          >
+        </li>
+        <li>
+          <a class="nav-link" href="#JavaScript_and_Java"
+            >JavaScript and Java</a
+          >
+        </li>
+        <li><a class="nav-link" href="#Hello_world">Hello world</a></li>
+        <li><a class="nav-link" href="#Variables">Variables</a></li>
+        <li>
+          <a class="nav-link" href="#Declaring_Variables"
+            >Declaring variables</a
+          >
+        </li>
+        <li><a class="nav-link" href="#while_statement">while statement</a></li>
+        <li></li>
+      </ol>
+    </nav>
```

Comment:
Congratulations on using semantic tags like the nav tag. You can consider nesting your nav tags within the header tags.

<header>
    <nav id="navbar">
      <header class="js_doc">JS Documentation</header>
      <ol>
        <li><a class="nav-link" href="#Introduction">Introduction</a></li>
        <li>
          <a class="nav-link" href="#What_you_should_already_know"
            >What you should already know</a
          >
        </li>
        <li>
          <a class="nav-link" href="#JavaScript_and_Java"
            >JavaScript and Java</a
          >
        </li>
        <li><a class="nav-link" href="#Hello_world">Hello world</a></li>
        <li><a class="nav-link" href="#Variables">Variables</a></li>
        <li>
          <a class="nav-link" href="#Declaring_Variables"
            >Declaring variables</a
          >
        </li>
        <li><a class="nav-link" href="#while_statement">while statement</a></li>
        <li></li>
      </ol>
    </nav>
</header>

Reply:
[ ] Kindly phrase your comments politely, as suggestions instead of orders. example:`"I think it would be better to use semantic tags"`
[ ] Kindly include an explanation, a "why", for each requested change. For example:`Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible

Reply:
- [ ] As suggested by the preview reviewer. Kindly make sure to phrase your `comment` politely and provide some context on why the change you're requesting is necessary. This can help the student 👨‍🎓 understand the reasoning behind the change and prevent them from making similar mistakes in the future. By explaining the importance of the change, the student 👨‍🎓 can also learn more effectively and improve their coding skills. Keep up the good work!